src/libraries/Microsoft.Extensions.Hosting/src/HostOptions.cs

```diff
@@ -17,6 +17,12 @@ public class HostOptions
         /// </summary>
         public TimeSpan ShutdownTimeout { get; set; } = TimeSpan.FromSeconds(5);
 
+        /// <summary>
+        /// The behavior the <see cref="IHost"/> will follow when any of
+        /// its <see cref="BackgroundService"/> instances throw an unhandled exception.
+        /// </summary>
+        public BackgroundServiceExceptionBehavior BackgroundServiceExceptionBehavior { get; set; }
```

Comment:
I believe in the API design review we agreed to make `StopHost` the default behavior.

Reply:
When I re-watched the video and the final decision was to leave it as `Ignored` for now as that is the current behavior, and then introduce the breaking change as part of .NET 7. I could be wrong, but that is what I saw/heard...happy to change it - if everyone agrees?

Reply:
Ah, I do see that it was agreed to change the default to `StopHost` and not update the templates.